api.planx.uk/modules/auth/passport.ts

```diff
@@ -14,7 +19,29 @@ export default async (): Promise<Authenticator> => {
   const customPassport = new passport.Passport();
 
   // instantiate Microsoft OIDC client, and use it to build the related strategy
-  const microsoftIssuer = await Issuer.discover(MICROSOFT_OPENID_CONFIG_URL);
+  // we also keep said config as a fixture to enable offline local development
```

Comment:
this change was literally just to allow me to run the API for development purposes (i.e. do `pnpm dev`) while I was on a train with no internet, so it doesn't necessarily need to make it into the codebase, but could be useful for others?

afaik fetching the config for the Microsoft OIDC stuff is the only call the code makes when spinning up, so it was possible to run `pnpm dev` to completion while offline before we added that SSO option, but now it isn't (without this change)

Reply:
I'd say no harm in keeping this around! Fixture may additionally come in handy for future E2E test mocks if we ever want Microsoft SSO coverage?

Reply:
@jessicamcinchak ok great, that makes sense as well, would be good to have this tested!